src/language/interpreter.rs

```diff
@@ -1091,17 +1091,23 @@ mod tests {
     /// Creates a benchmark test for the interpreter
```

Comment:
This should be something like "Creates a benchmark and a test for the interpreter", because now we're creating both things!